### PR TITLE
guide: bootstrap guide book

### PR DESCRIPTION
**Pull request by joyeecheung**
This book currently looks like this:

localhost_4000_ (2)

I can add a workflow to make it deploy to github pages later (probably would be similar to https://github.com/DjangoGirls/tutorial/blob/master/.github/workflows/deploy.yml)

Fixes: https://github.com/nodejs/package-examples/issues/5

### CODE REVIEW

**Comment by joyeecheung** [comment]
cc @nodejs/loaders @nodejs/package-maintenance

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
-# Developer's Certificate of Origin 1.1
+# Contributing guide
+
+## Building the guide locally
+
+```bash
+# Clone this repository first, and then switch to the project directory
+cd package-examples
+cd guide
+npm ci
```
Comment (bjohansebas) [inline CONTRIBUTING.md:9]: This command doesn't exist yet

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+npm run build  # Build the guide book in _book
+npm start      # Run a development server that host the built guide
+```
```
Comment (bjohansebas) [inline CONTRIBUTING.md:12]: Just running `npm start` was enough for me.

**Comment by bjohansebas** [comment]
> There is actually a semi official way to do this now I think. Check out this: https://github.com/pkgjs/statusboard/blob/main/.github/workflows/build.yml#L52-L54
> 
> I think to get it to work we also had to change a setting in settings/pages.

Yeah, it should work by just copying and pasting

File: CONTRIBUTING.md (hunk 2 above)
Comment (joyeecheung) [reply CONTRIBUTING.md:12]: The build command is not necessary for starting the dev server, but is useful on its own if a dev server is not necessary.

**Comment by joyeecheung** [comment]
> There is actually a semi official way to do this now I think. Check out this:

Yes, that's the same actions used by https://github.com/DjangoGirls/tutorial/blob/master/.github/workflows/deploy.yml that I was referencing ;) just that they can be reused for this guide since they are also using Honkit (actions/upload-pages-artifact@v3 + actions/deploy-pages@v4)

**Comment by wesleytodd** [comment]
> that's the same actions used by

Ah sorry, I didn't click through to that link. My bad.

**Comment by charlieforward9** [comment]
this was quick! Ill test it out once I see a link. thanks yall!

**Pull request merged**